HangPerson/HangPerson/main.m

```diff
@@ -12,7 +12,61 @@ int main(int argc, const char * argv[]) {
     @autoreleasepool {
         
         // code goes here...
-
+        char word[5] = "pasta";
+        char dashes[5] = "_____";
+        char input;
+        char temp;
+        int guesses = 7; //Maximum tries.
+        int i;
+        int j;
+        int swaps = 5;
+        
+        
+        printf("Lets play a Hangman.\n");
+        printf("I'm thinking of a word. Can you guess it?\n");
+        
+        for(i = 0; i < guesses; i++ ) {
```

Comment:
The condition here shouldn't be based on the number of guess because you could guess the word in 4 guesses. A while loop would be better in this situation. Something along the lines of: 

```c
int won = 0;
int lost = 0;
while (won == 0 && lost == 0) { 
  // play game
}
```